### PR TITLE
Add option to disable ES SSL cert verification

### PR DESCRIPTION
**Pull request by marcinc**

### CODE REVIEW

File: kibana.yml [hunk 1]
```diff
@@ -48,7 +48,7 @@
 # elasticsearch.ssl.ca: /path/to/your/CA.pem
 
 # To disregard the validity of SSL certificates, change this setting’s value to false.
-# elasticsearch.ssl.verify: true
+elasticsearch.ssl.verify: false
```
Comment (vaijab) [inline kibana.yml]: Maybe make it configurable? It is probably not a good default.

**Review by vaijab — CHANGES_REQUESTED** [review]

**Comment by vaijab** [comment]
I believe kibana config can read env vars directly.

**Comment by marcinc** [comment]
@vaijab Unfortunately not. Their products are very inconsistent. Added simple run script to substitute cert verification with ENV variable.

**Review by vaijab — APPROVED** [review]
LGTM

**Pull request merged**